master/master.cfg

```diff
@@ -379,6 +379,7 @@ c["www"] = dict(
         },
     },
     plugins=dict(waterfall_view={}, console_view={}, grid_view={}),
+    ws_ping_interval=30,
```

Comment:
Maybe add a comment to https://bugs.python.org/issue41701

Reply:
I still need to figure out how to update the server